vyper/ast/parse.py

```diff
@@ -133,11 +130,8 @@ def annotate_python_ast(
         The AST to be annotated and optimized.
     vyper_source: str
         The original vyper source code
-    loop_var_annotations: dict
-        A mapping of line numbers of `For` nodes to the tokens of the type
-        annotation of the iterator extracted during pre-parsing.
-    modification_offsets : dict
-        A mapping of class names to their original class types.
+    pre_parse_result: PreParseResult
```

Comment:
nit
```suggestion
    pre_parser: PreParser
```

Reply:
as discussed on TG, we will look into the renaming separately alongside moving `pre_parse()` within `PreParser`.